jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/ImageDebugSupport.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.execution.sam
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.intellij.openapi.extensions.ExtensionPointName
+import com.intellij.openapi.util.io.FileUtil
+import software.aws.toolkits.jetbrains.core.utils.buildList
+import java.util.UUID
+
+interface ImageDebugSupport : SamDebugSupport<ImageTemplateRunSettings> {
+    val id: String
+
+    /**
+     * The primary language id, used to find the correct builder for the runtime
+     */
+    val languageId: String
```

Comment:
To be able to build properly, it seems like we can either:
1. Store the runtime type itself
2. Store the runtime type id
3. store a language id associated with the runtime
I think storing the language id, while being a bit strange, seems the most reasonable so we don't have a dependency on runtime type. Thoughts?

Reply:
In the JVM world, if we have kotlin and java,

would we use langaugeId Java?

Reply:
As is yes because it's not actually target. I think we need to invert this to put the additional information in our own `Runtime` long term